lua/rose-pine/theme.lua

```diff
@@ -18,6 +18,7 @@ function M.get(config)
 	theme = {
 		ColorColumn = { bg = p.highlight_high },
 		Conceal = { bg = p.none },
+		CurSearch = { bg = p.rose, fg = p.pine },
```

Comment:
We should make this consistent with our `IncSearch` highlight 😌

```suggestion
		CurSearch = { link = 'IncSearch' },
```

Reply:
ok! Here's how that looks:

<img width="122" alt="image" src="https://user-images.githubusercontent.com/1973/167174006-77f9d2a7-5c4e-4e10-a888-e540c707fd2a.png">
<img width="195" alt="image" src="https://user-images.githubusercontent.com/1973/167174046-1fac233b-76ef-486b-8112-5a243c0e030f.png">

Reply:
Updated the PR with that